frontend/appflowy_tauri/src/appflowy_app/utils/slate/hotkey.ts

```diff
@@ -14,9 +16,72 @@ const HOTKEYS: Record<string, string> = {
 export function triggerHotkey(event: React.KeyboardEvent<HTMLDivElement>, editor: Editor) {
   for (const hotkey in HOTKEYS) {
     if (isHotkey(hotkey, event)) {
-      event.preventDefault()
-      const format = HOTKEYS[hotkey]
-      toggleFormat(editor, format)
+      event.preventDefault();
+      const format = HOTKEYS[hotkey];
+      toggleFormat(editor, format);
     }
   }
-}
\ No newline at end of file
+}
+
+export function canHandleEnterKey(event: React.KeyboardEvent<HTMLDivElement>, editor: Editor) {
+  const isEnter = event.key === 'Enter';
+  if (!isEnter || !editor.selection) {
+    return false;
+  }
+  return true;
+}
+
+export function canHandleBackspaceKey(event: React.KeyboardEvent<HTMLDivElement>, editor: Editor) {
+  const isBackspaceKey = event.key === 'Backspace';
+  if (!isBackspaceKey || !editor.selection) {
+    return false;
+  }
+  // It should be handled if the selection is collapsed and the cursor is at the beginning of the block
+  const { anchor } = editor.selection;
+  const isCollapsed = Range.isCollapsed(editor.selection);
+  return isCollapsed && anchor.offset === 0 && anchor.path.toString() === '0,0';
```

Comment:
use `editor.selection` many times here. how about assign it to a local value.

```ts
const { selection } = editor;
if (!isBackspaceKey || selection == null) {
  return false;
}
const { anchor } = selection;
// ....
```

I'm not sure the grammar is correct.